src/App.jsx

```diff
@@ -14,70 +17,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
```

Comment:
```suggestion
  const [selectedGood, setGood] = useState(goods[goods.indexOf('Jam')] || good[0]);
```

to avoid bugs if won't be 'Jam' in array